bin/maf/metadata_dir

```diff
@@ -3,48 +3,18 @@
 import os
```

Comment:
`metadata_dir` needs to be added to the `setup.py` to get it in users' path.

Reply:
Yeah, I know .. I also know you chose to leave it out when you set up the executables before, so I figured maybe there was a reason.